OpenMarket/OpenMarket/View/View+Extension.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  view+extension.swift
+//  OpenMarket
+//
+//  Created by 우롱차, Donnie on 2022/05/18.
+//
+
+import UIKit
+
+extension UIView {
+    static var identifier: String {
+        return String(describing: self)
+    }
+}
+
+extension UIStackView {
+    func addLastBehind(view: UIView) {
+        let lastViewCount = self.arrangedSubviews.count
+        if lastViewCount <= 1 {
+            self.insertArrangedSubview(view, at: 0)
+        } else {
+            self.insertArrangedSubview(view, at: lastViewCount - 1)
+        }
```

Comment:
개인 취향에 따라 조금 갈릴 것 같긴 한데.. 삼항 연산자를 활용해 좀 더 간단하게 표현할 수도 있을 것 같아요.
```suggestion
        let index = lastViewCount <= 1 ? 0 : lastViewCount - 1
        self.insertArrangedSubview(view, at: index)
```

Reply:
삼항연산자를 많이 써버릇안해서 그러네요! 다음부터 많이 써볼게요

Reply:
삼항 연산자는 사람마다 취향이 많이 갈리는 부분이라 굳이 많이 써보려고 하시진 않아도 됩니다 😅
이런 방법도 있다 정도만 알아주심 좋을 것 같아요~